ecosystem/indexer-grpc/indexer-grpc-data-service/src/metrics.rs

```diff
@@ -68,10 +68,11 @@ pub static PROCESSED_BATCH_SIZE: Lazy<IntGaugeVec> = Lazy::new(|| {
 });
 
 /// Count of connections that data service has established.
-pub static CONNECTION_COUNT: Lazy<IntCounter> = Lazy::new(|| {
-    register_int_counter!(
+pub static CONNECTION_COUNT: Lazy<IntCounterVec> = Lazy::new(|| {
+    register_int_counter_vec!(
         "indexer_grpc_data_service_connection_count",
```

Comment:
you may want to rename the metric name otherwise the schema is conflicting between scratchnet and mainnet.